spring-cloud-function-web/src/main/java/org/springframework/cloud/function/web/RestConfiguration.java

```diff
@@ -89,20 +93,21 @@ private FunctionInvokingHandler(Function<Flux<String>, Flux<String>> function) {
 			this.function = function;
 		}
 
-		private  Response<Publisher<String>> handleText(Request request) {
+		private Mono<ServerResponse> handleText(ServerRequest request) {
 			Flux<String> input = request.body(toFlux(String.class));
 			Publisher<String> output = this.function.apply(input);
-			return Response.ok().body(fromPublisher(output, String.class));
+			return ServerResponse.ok().body(fromPublisher(output, String.class));
 		}
 	}
 
-	private static class LifecycleAwareHttpServer implements InitializingBean, DisposableBean {
+	private static class LifecycleAwareHttpServer
+			implements InitializingBean, DisposableBean {
 
 		private final HttpHandler handler;
 
 		private final int port;
 
-		private volatile HttpServer server;
+		private volatile NettyContext server;
```

Comment:
I don't know how stable it is. I don't mind either way. Just wanted something that works.
I figure we'll migrate to spring boot stack eventually anyway.